whosonfirst/view/shapefile.js

```diff
@@ -23,13 +23,24 @@ module.exports = (feat, params) => {
   // exportable filters
   if (!filters.exportable(feat, spr)) return null
 
+  // logic for placetype local property
```

Comment:
As the feature.js logic was getting complicated and hard to read, I moved it to shapefile.js. 

If that's OK, great. Else I'll need help porting this logic back to feature.js

@missinglink